localization/english/wc_gui_l_english.yml

```diff
@@ -0,0 +1,17 @@
+﻿l_english:
+ CV_HAS_NO_DIPLOMACY:0 "Has no ongoing diplomacy"
+ CV_ALLIANCE:0 "Alliance"
+ CV_HORDE:0 "Horde"
+ CV_MEMBERS:0 "Members #weak ([GetPlayer.MakeScope.Var('gui_pol_faction_member_number').GetValue|V|0])#!"
+ CV_LEADER:0 "[leader_wo_union|E]"
+ CV_UNION_SOLDIERS:0 "Military Strength: [GetPlayer.MakeScope.Var('gui_pol_faction_strength').GetValue|V|0]"
+ 
+ POL_FACTIONS_WINDOW_TITLE:0 "Unions"
+ HUD_POL_FACTIONS:0 "Unions"
```

Comment:
Why not use " [union|E] " here instead of "Union", just in case something changes eventually?

Reply:
Changed in 74299d9bf6e6decbc1a602c43b19b8242bcad869.